src/traces/contour/attributes.js

```diff
@@ -239,9 +239,17 @@ module.exports = extendFlat({
                 'Has no effect if `contours.coloring` is set to *lines*.'
             ].join(' ')
         }),
-        width: extendFlat({}, scatterLineAttrs.width, {
-            editType: 'style+colorbars'
-        }),
+        width: {
+            valType: 'number',
+            min: 0,
+            role: 'style',
+            editType: 'style+colorbars',
+            description: [
+                'Sets the contour line width in (in px)',
+                'Defaults to *0.5* when `contours.type` is *levels*.',
+                'Defaults to *2* when `contour.type` is *constraint*.'
+            ].join(' ')
```

Comment:
Here's the corresponding default logic:

https://github.com/plotly/plotly.js/blob/1dee68b7fdd768c5dbc4b3644d4bcbd15f8e591a/src/traces/contour/style_defaults.js#L25

reused by `contourcarpet` and `histogram2dcontour`

https://github.com/plotly/plotly.js/blob/1dee68b7fdd768c5dbc4b3644d4bcbd15f8e591a/src/traces/contour/constraint_defaults.js#L46

reused by `contourcarpet`